mcan/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Tagging in git follows a pattern: `mcan/<version>`.
 
 ## [Unreleased]
+- Add `Can::aux::shutdown` to safely shutdown the bus (#45)
```

Comment:
Don't forget to update the changelog with the updated terminology and function names.

Reply:
Fixed.